Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardEditView.swift

```diff
@@ -50,9 +52,13 @@ struct CreditCardEditView: View {
                 borderColor: borderColor,
                 alertDetails: viewModel.removeButtonDetails
             )
+            Spacer()
         }
         .padding(.top, 20)
-        Spacer()
+        .toast(
+            toastView: ToastView(messageType: .removedCard, isShowing: $viewModel.shouldShowToast),
+            isShowing: $viewModel.shouldShowToast
+        )
```

Comment:
When we remove the card then the view is dismissed so in theory you won't see this toast show up here unless we attach it to the list view
cc @yoanarios

Reply:
ok I will move to ListView do we have events already when a card is saved or removed?

Reply:
No particular events but we can add a delegate or closure to the list view and pass it along view model of edit card for callbacks